.buildkite/scripts/buildx.sh

```diff
@@ -4,11 +4,9 @@ set +x
 
 BUILDPLATFORM=${BUILDPLATFORM:-"linux/amd64,linux/arm64"}
 
-#docker run --privileged --rm tonistiigi/binfmt --install all
-docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
qq: what's the reason for this deletion?

Reply:
this is not needed:

- it registers the QEMU binary format emulation with the kernel (binfmt_misc).
- is used primarily for enabling QEMU-based emulation for different architectures

tonistiigi/binfmt automatically installs and configures QEMU for multiple architectures in a Docker-friendly way and handles registration more flexibly than multiarch/qemu-user-static.

Reply:
Awesome, thanks for the clarification 🙏